_tut/optional.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title:  "Optional"
+section: "optics"
+scaladoc: "http://julien-truffaut.github.io/Monocle/api/#monocle.POptional"
+pageSource: "https://raw.githubusercontent.com/julien-truffaut/Monocle/master/docs/src/main/tut/optional.md"
+---
+# Optional
+
+An `Optional` is an Optic used to zoom inside a `Product`, e.g. `case class`, `Tuple`, `HList` or even `Map`.
+Unlike the `Lens`, the element that the `Optional` focus on may do not exist.
+
+`Optionals` have two type parameters generally called `S` and `A`: `Optional[S, A]` where `S` represents the `Product` and `A` an optional element inside of `S`.
+
+Let's take a simple list with integers.
+
+We can create an `Optional[List[Int], Int]` which zoom from a `List[Int]` to its potential head by supplying a pair of functions:
+
+*   `getOption: List[Int] => Option[Int]`
+*   `set: Int => List[Int] => List[Int]`
+
+```scala
+import monocle.Optional
+val _head = Optional[List[Int], Int] {
```

Comment:
I would prefer not to use `_` as a prefix. Historically, we used this prefix but we started to get rid of it.